### PR TITLE
Updates Events to Pull from AirTable rather than Google Sheet

### LINKED ISSUE #937: Move Events from GS to AIRtable
### Expected Behavior

We get events from airtable 

### Current Behavior

they exist in GSheets

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Fetch Events from Community Events AirTable

Removes TableTop dependency

### Tested

Added tests for the server side event processors. and confirmed page loads with events that match those in airtable. 

### Other changes

Fixes Sentry calls to be sentry.CaptureEvent rather than exception as exception should only be used for Real errors. 

### Related issues

- Fixes #937 

### Backwards compatibility

Events stil load

### CODE REVIEW

File: packages/web/fullstack/EventProps.ts [hunk 1]
```diff
@@ -3,10 +3,9 @@ export interface EventProps {
   name: string
   startDate: string
   location: string
-  link: string
+  link?: string
```
Comment (aaronmgdr) [inline packages/web/fullstack/EventProps.ts:6]: not all events from airtable will have a link

File: packages/web/fullstack/EventProps.ts [hunk 2]
```diff
@@ -3,10 +3,9 @@ export interface EventProps {
...
   celoHosted: boolean
   celoSpeaking: boolean
-  celoAttending: boolean
```
Comment (aaronmgdr) [inline packages/web/fullstack/EventProps.ts:9]: this hasnt been used for a while now

File: packages/web/package.json [hunk 3]
```diff
@@ -68,7 +68,6 @@
     "shuffle-seed": "^1.1.6",
     "stickyfilljs": "^2.0.5",
     "svgs": "^4.1.0",
-    "tabletop": "^1.5.2",
```
Comment (aaronmgdr) [inline packages/web/package.json:71]: bye

File: packages/web/server/airtable.ts [hunk 4]
```diff
@@ -11,3 +11,21 @@ export default function airtableInit(baseID: string) {
   }
   return airTableSingleton.base(baseID)
 }
+
+export interface AirRecord<Fields> {
```
Comment (aaronmgdr) [inline packages/web/server/airtable.ts:15]: representation of what the airtable API returns.

**Review by cmcewen — APPROVED** [review]

**Pull request merged**